src/gui/tray/Window.qml

```diff
@@ -167,6 +181,10 @@ Window {
                                 radius: Style.currentAccountButtonRadius
                             }
 
+                            Accessible.role: PopupMenu
+                            Accessible.name: qsTr("Account menu")

```

Comment:
Ah ok – "Current account" and "Account menu" are 2 different things. Then ignore my very first review.

Maybe I would say to call this "Account switcher and settings" instead to clarify?